mlir/include/mlir/Dialect/Rock/IR/RockOps.td

```diff
@@ -363,10 +363,10 @@ def Rock_GridwiseGemmV2Op :
                    StoreMethodAttr:$storeMethod,
                    I32Attr:$blockSize,
                    I32Attr:$gridSize,
-                   Rock_XdlopsGemmParamsAttr:$params)> {
+                   RockAccelTuningParamAttrInterface:$params)> {
   let summary = "Gridwise GEMM V2";
```

Comment:
Heck, if we're doing the big renaming kick, maybe this could be named `gridwise_gemm_accel` and we'd have `blockwise_gemm_accel` as well?